lib/segment/tests/integration/byte_storage_hnsw_test.rs

```diff
@@ -82,14 +82,14 @@ fn compare_search_result(result_a: &[Vec<ScoredPointOffset>], result_b: &[Vec<Sc
 }
 
 #[rstest]
-#[case::nearest(QueryVariant::Nearest, VectorStorageDatatype::Uint8, 32, 10)]
-#[case::nearest(QueryVariant::Nearest, VectorStorageDatatype::Float16, 32, 10)]
-#[case::discovery(QueryVariant::Discovery, VectorStorageDatatype::Uint8, 128, 20)]
+#[case::nearest(QueryVariant::Nearest, VectorStorageDatatype::Uint8, 32, 20)]
```

Comment:
I decreased required acc here. Don't know actually why. But after updating size estimation in HNSW, this test did low cardinality searches instead of HNSW. I changed the filter generation to restore search over HNSW but accuracy decreases after it

Reply:
I propose to change `full_scan_threshold` to 0, so it would be fair for the HNSW

Reply:
Reverted this change and set `full_scan_threshold = 0` instead